ocfl-java-itest/src/test/java/io/ocfl/itest/LoadITest.java

```diff
@@ -445,10 +383,22 @@ private OcflRepository createFsRepo() {
     }
 
     private MutableOcflRepository createS3Repo() {
-        var s3Client = S3Client.builder()
-                .region(Region.US_EAST_2)
-                .httpClientBuilder(ApacheHttpClient.builder())
-                .build();
+        //        var s3Client = S3AsyncClient.builder()
+        //                .region(Region.US_EAST_2)
+        //                .httpClientBuilder(NettyNioAsyncHttpClient.builder()
+        //                        .connectionAcquisitionTimeout(Duration.ofSeconds(60))
+        //                        .writeTimeout(Duration.ofSeconds(0))
+        //                        .readTimeout(Duration.ofSeconds(0))
+        //                        .maxConcurrency(100))
+        //                .build();
+        //        var transferManager = S3TransferManager.builder()
+        //                .s3Client(MultipartS3AsyncClient.create(
+        //                        s3Client, MultipartConfiguration.builder().build()))
+        //                .build();
```

Comment:
@pwinckles -- Do you want to save this commented-out code or can it be removed?

Reply:
This class I mostly use for experimentation, so, yeah, I had planned on keeping that code.